client/app/billing/payment/method/add/index.html

```diff
@@ -0,0 +1,139 @@
+                            <div data-ng-if="$ctrl.model.selectedPaymentMethodType">
+                                <div data-ng-if="$ctrl.model.selectedPaymentMethodType.paymentType === $ctrl.OVH_PAYMENT_METHOD_TYPE.CREDIT_CARD">
+                                    <p data-ng-if="$ctrl.currentUser.ovhSubsidiary === 'FR'"
+                                       data-translate="billing_payment_method_add_credit_card_selected_explain"
+                                       data-translate-values="{ amount: $ctrl.creditCardFootprintAmount }">
+                                    </p>
+                                    <p data-ng-if="$ctrl.model.selectedPaymentMethodType.integration !== $ctrl.OVH_PAYMENT_METHOD_INTEGRATION_TYPE.IFRAME_VANTIV && $ctrl.currentUser.ovhSubsidiary !== 'FR'"
+                                       data-translate="billing_payment_method_add_credit_card_selected_explain_generic">
+                                    </p>
+                                    <p data-translate="billing_payment_method_add_credit_card_selected"></p>
+                                    <p data-ng-if="$ctrl.model.selectedPaymentMethodType.isLegacy() && $ctrl.model.selectedPaymentMethodType.integration !== $ctrl.OVH_PAYMENT_METHOD_INTEGRATION_TYPE.IFRAME_VANTIV"
+                                       data-translate="billing_payment_method_add_selected_common_redirect">
+                                    </p>
+                                </div>
+                                <p data-ng-if="$ctrl.model.selectedPaymentMethodType.paymentType === $ctrl.OVH_PAYMENT_METHOD_TYPE.BANK_ACCOUNT">
+                                    <span class="d-block"
+                                          data-translate="billing_payment_method_add_selected_common_redirect"
+                                          data-ng-if="$ctrl.model.selectedPaymentMethodType.isLegacy()">
+                                    </span>
+                                    <span class="d-block"
+                                          data-translate="billing_payment_method_add_bank_account_selected"
+                                          data-ng-if="!$ctrl.model.selectedPaymentMethodType.isLegacy()">
+                                    </span>
+                                </p>
+                                <p data-ng-if="$ctrl.model.selectedPaymentMethodType.paymentType === $ctrl.OVH_PAYMENT_METHOD_TYPE.PAYPAL"
+                                   data-translate="billing_payment_method_add_selected_paypal_redirect">
+                                </p>
+                            </div>
+                        </explanation-text>
+                    </ovh-payment-method-register>
+
+                    <ovh-payment-method-integration
+                        data-ng-if="$ctrl.isLastStep('paymentMethodType')"
+                        data-on-initialized="$ctrl.onPaymentMethodIntegrationInitialized.bind($ctrl)"
```

Comment:
Why do you need to bind ? If you're binding is a `&` it will not be necessary I think

Reply:
Without I lose the reference of my controller in my callback